src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -358,6 +373,34 @@ namespace winrt::TerminalApp::implementation
         if (_startupState == StartupState::NotInitialized)
         {
             _startupState = StartupState::InStartup;
+
+            // If the user selected to save their tab layout, we are the first
+            // window opened, and wt was not run with any other arguments, then
+            // we should use the saved settings.
+            auto firstActionIsDefault = [](ActionAndArgs action) {
+                if (action.Action() != ShortcutAction::NewTab)
+                {
+                    return false;
+                }
+
+                // If no commands were given, we will have default args
+                if (const auto args = action.Args().try_as<NewTabArgs>())
+                {
+                    NewTerminalArgs defaultArgs{};
+                    return args.TerminalArgs() == nullptr || args.TerminalArgs().Equals(defaultArgs);
```

Comment:
technically `wt nt` will trigger this, but I don't know if i care :smile:

Reply:
in #11083 I think this is partially ameliorated in that when we open multiple windows we actually check if any arguments at all were provided, and if so we make a new window instead of putting the saved layout in the current window.